regex/noncap-building.regex

```diff
@@ -1 +1 @@
-(?:[A-Z\-]*)
+(?:[A-Z\-]{0,8})
```

Comment:
I'm assuming you choose 8 as the maximum length based on the building codes I entered, right?